### ISSUE TITLE
[bug] Endpoint containing hash return 404 error

### ISSUE THREAD

**Issue by tseropian** [mockoon/mockoon#1118]
**Describe the bug**

Endpoints with a _hash_ on the path are returned as `404` errors.

**To Reproduce**

- Create an endpoint with an hash (`/hello#world`)
- This can also be reproduced using some mock samples provided by Mockoon (for example [AWS Comprehend](https://mockoon.com/mock-samples/amazonawscom-comprehend/)). Endpoint are like the following: `#X-Amz-Target=Comprehend_20171127.BatchDetectDominantLanguage`

**Mockoon version:**

4.0.0

**OS / OS version:**

Mac OS Ventura and Amazon Linux (for the CLI)

**Comment by 255kb** [comment]
We use Express for the routing, and you can perfectly define a route containing hashtags. However, the hashtag is not part of the URL and is not sent to the server by most tools (curl, etc.). So, it's something that is not recommended.

That being said, the problem you have is not really linked to this but rather to the AWS APIs OpenAPI specifications and how we parse them.
There is an open issue AWS APIs [here](https://github.com/mockoon/mock-samples/issues/8). 
Long story short, there is only one endpoint in the whole API, but the hashtag in describing the use of a different **header** every time: `#X-Amz-Target=Comprehend_20171127.BatchDetectDominantLanguage` (where `X-Amz-Target` is a header). 

I will close this in favor of the other issue.
